inspirehep/modules/records/receivers.py

```diff
@@ -170,20 +175,106 @@ def populate_bookautocomplete(sender, json, *args, **kwargs):
     input_values.extend(titles)
     input_values = [el for el in input_values if el]
 
-    ref = get_value(json, 'self.$ref')
+    record = get_value(json, 'self.$ref', '')
 
     json.update({
         'bookautocomplete': {
             'input': input_values,
             'payload': {
                 'authors': authors,
```

Comment:
@michamos I need more indications for `bookautocomplete`. Currently it has no `output` and we need to decide on it and what should be in the payload, right @harunurhan ?

Reply:
input seems fine, output for `parent_isbn` should be isbn.